src/Misc/expressionFunc/hashFiles/src/hashFiles.ts

```diff
@@ -52,12 +52,13 @@ async function run(): Promise<void> {
   }
 }
 
-run()
-  .then(out => {
+;(async () => {
```

Comment:
eslint was erroring on this promise chain, and wanted it converted to an async/await for node 20, hence the anonymous function call